README.md

```diff
@@ -320,6 +320,13 @@ to connect to one it will try the next in the list, and repeat until a
 successful connection is made. In the event of a disconnect, the agent will
 retry that connection before trying a new broker.
 
+**master-uris**
```

Comment:
Does this need to be optional for backwards compatibility?
Should pxp agent still allow puppet runs when it's misconfigured so there is a good way to fix it remotely?